backend/domain/creation.go

```diff
@@ -4,3 +4,16 @@ type TextReq struct {
 	Text   string `json:"text" validate:"required"`
 	Action string `json:"action"` // action: improve, summary, extend, shorten, etc.
 }
+
+// FIM (Fill in Middle) tokens
+const (
+	FIMPrefix = "<fim_prefix>"
+	FIMSuffix = "<fim_suffix>"
+	FIMMiddle = "<fim_middle>"
+)
+
+type CompleteReq struct {
```

Comment:
CompleteReq结构体的注释可以更详细一些，说明prefix和suffix在FIM场景下的具体作用。

```suggestion
type CompleteReq struct {
    // Prefix: 光标前的文本内容，在FIM中作为上下文提供给模型
    Prefix string `json:"prefix,omitempty"`
    // Suffix: 光标后的文本内容，在FIM中作为后续上下文提供给模型
    Suffix string `json:"suffix,omitempty"`
}